verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -166,16 +184,27 @@ void IndexingFactsTreeExtractor::ExtractModuleHeader(
     return;
   }
 
-  const std::vector<verible::TreeSearchMatch> port_names =
-      FindAllModulePortDeclarations(*port_list);
+  Visit(*port_list);
+}
 
-  for (const verible::TreeSearchMatch& port : port_names) {
+void IndexingFactsTreeExtractor::ExtractModulePort(
+    const verible::SyntaxTreeNode& module_port_node) {
+  const auto tag = static_cast<verilog::NodeEnum>(module_port_node.Tag().tag);
+
+  if (tag == NodeEnum::kPortDeclaration) {
```

Comment:
// summarize with a comment that shows the difference between ANSI (with direction/type) and non-ANSI style port.